pkgs/tools/security/notary/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchgit, buildGoPackage, git, libtool }:
+
+buildGoPackage rec {
+  name = "notary-${version}";
+  version = "0.4.3";
+  gitcommit = "9211198";
+
+  src = fetchgit {
+    url = "https://github.com/theupdateframework/notary";
+    rev = "refs/tags/v${version}";
+    sha256 = "1max6rgangmw892l8zpijhx5bj9zllqim457k1dq7vf1mzybihr6";
+    leaveDotGit = true;
```

Comment:
```
output path '/nix/store/g0x3f272dhwns82v2ygaa3w72y2gbhkp-notary' has r:sha256 hash '1j75973a6vciwcrqxbzvk6smr5s502dsa43723g3k9f33nz8lyhz' when '1max6rgangmw892l8zpijhx5bj9zllqim457k1dq7vf1mzybihr6' was expected
```
Looks like the checksum of `fetchgit` changed or my systems creates a different one then yours.
Can we get rid of `leaveDotGit`?

Reply:
It should be possible to set `make GITCOMMIT=<hash> GITUNTRACKEDCHANGES= ` and get rid of the git repository:
https://github.com/theupdateframework/notary/blob/master/Makefile#L8